packages/render-fidelity-tools/package.json

```diff
@@ -23,8 +23,8 @@
   },
   "dependencies": {
     "@actions/core": "^1.2.4",
-    "@babylonjs/core": "^5.0.0-alpha.31",
-    "@babylonjs/loaders": "^5.0.0-alpha.31",
+    "@babylonjs/core": "*5.0.0-alpha.31",
+    "@babylonjs/loaders": "*5.0.0-alpha.31",
```

Comment:
This tried to update to a newer version when I added our simple-dropzone dependency, and broke something. I pinned the  version to fix it, but we'll have to deal with this when we update eventually.